tests/endpoint_decorators_test.py

```diff
@@ -42,3 +42,38 @@ def test_log_headers(*args, **kwargs):
                 "Authorization: REDACTED\n"
                 "authorization: REDACTED\n"
                 "SafeStuff: This better be logged this way")
+
+    def test_logbodies_errors_on_invalid_json(self):
+        """
+        """
```

Comment:
Why is this empty? I am guessing you wanted to write something in here. Like what this tests.